src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -130,6 +130,19 @@
     <value>Invalid file name: It must contain only {0}.</value>
     <comment>{0} - The regular expression that the file name must match</comment>
   </data>
+  <data name="AAPTManifestError" xml:space="preserve">
+    <value>{0}. This error is likely caused by an issue with the AndroidManifest.xml file or an Android manifest generation attribute in a source code file.</value>
```

Comment:
@dellis1972: while attempting to write the [above commit message](https://github.com/xamarin/xamarin-android/pull/4614#issuecomment-674600906), I turn my attention to this error message.  I have two philosophical concerns with it:

1. We emit only one error message, and it's a (potentially) very *long* error message at that.  While the error message is lined wrapped, e.g. [this VS error list image](https://blogimages.builttoroam.com/OT-Visual-Studio-2019-Makes-Build-Output_D884/image_3.png), a message of this length may look "weird" in context.

    Is there a better way?

    @brendanzagaeski: Did we ever figure out how to sanely do multi-line error messages that integrate nicely in VS?

2. The message mentions "the AndroidManifest.xml" file.  Should we mention *which* `AndroidManifest.xml` file?

    Additionally, particularly in our new/current `manifestmerger.jar` world order (2c6f5cd1, f7ea4a3db003da7be85d34eff4813c6052368e90), there can be *multiple* `AndroidManifest.xml` files.  *Presumably* these other files won't be the cause of errors, but can we be absolutely certain?

    It may thus not be possible to *know* "which" `AndroidManifest.xml` file is the cause of the error either.

Reply:
We should only deal with "actionable" files I think. Pointing the user to one of the Intermediate files does not help them as if they edit the file it will just be overwritten.  

The error links to the main Manifest file so when the user double clicks on it I the IDE that is the one which will be opened.

Reply:
> @brendanzagaeski: Did we ever figure out how to sanely do multi-line error messages that integrate nicely in VS?

As far as I know, it's fine to use line breaks within the string for a single call to `LogCodedWarning()` or `LogCodedError()` if desired. So for example, it would be OK to do something like:

```
<value>{0}.

This error is likely caused by an issue with the AndroidManifest.xml file or an Android manifest generation attribute in a source code file.</value>
```

This appears in the command line output like:

```
C:\Temp\Solution1\AndroidXAppSingleView1\Properties\AndroidManifest.xml : error APT2260: resource mipmap/ic_launche (aka com.companyname.androidappsingleview1:mipmap/ic_launche) not found.
C:\Temp\Solution1\AndroidXAppSingleView1\Properties\AndroidManifest.xml : error APT2260: 
C:\Temp\Solution1\AndroidXAppSingleView1\Properties\AndroidManifest.xml : error APT2260: This error is likely caused by an issue with the AndroidManifest.xml file or an Android manifest generation attribute in a source code file.
```

And it appears in Visual Studio like:

![image](https://user-images.githubusercontent.com/4934473/90431217-b1bea900-e096-11ea-8f6c-88e25e6dbde3.png)

Compared to no line breaks:

![image](https://user-images.githubusercontent.com/4934473/90431591-45907500-e097-11ea-941b-c76db97bc459.png)

Reply:
Please add line breaks.

Reply:
done